kubejs/server_scripts/enigmatica/kubejs/base/unification/unify_dyes.js

```diff
@@ -3,286 +3,286 @@ onEvent('recipes', (event) => {
     const id_prefix = 'enigmatica:base/unification/unify_dyes/';
 
     dyeSources.forEach((recipe) => {
-        botania_dye_pestle_mortar(event, recipe, id_prefix);
-        create_dye_milling(event, recipe, id_prefix);
-        ars_nouveau_dye_crushing(event, recipe, id_prefix);
-        immersiveengineering_dye_crusher(event, recipe, id_prefix);
-        //integrateddynamics_dye_squeezing(event, recipe, id_prefix);
-        mekanism_dye_enriching(event, recipe, id_prefix);
-        mekanism_pigment_extracting(event, recipe, id_prefix);
-        pedestals_dye_crushing(event, recipe, id_prefix);
-        thermal_dye_centrifuge(event, recipe, id_prefix);
-        atum_quern_milling(event, recipe, id_prefix);
-        shapeless_dye_crafting(event, recipe, id_prefix);
-        occultism_dye_crushing(event, recipe, id_prefix);
+        botania_dye_pestle_mortar(recipe, id_prefix);
+        create_dye_milling(recipe, id_prefix);
```

Comment:
Sorry for the suuuper late reply to this 😬 
Why remove event? I like that it's an explicit requirement for the function to work, rather than an implicitly defined variable.

Reply:
Sigh ignore me, I hadn't read the PR description properly. I get it now.